flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/plan/schema/MapRelDataType.scala

```diff
@@ -45,5 +45,4 @@ class MapRelDataType(
   override def hashCode(): Int = {
     Objects.hashCode(keyType, valueType)
   }
-
```

Comment:
the change is reverted ?

Reply:
Yes, the map type has nullability issue for its nested key and values, because currently we create the map type and ignore its key/value nullability (both of them are from the type information with nullability true as default).

Reply:
related issue: https://issues.apache.org/jira/browse/FLINK-20785